backend/capellacollab/core/database/models.py

```diff
@@ -1,12 +1,15 @@
 # SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
 # SPDX-License-Identifier: Apache-2.0
 
+# pylint: skip-file
```

Comment:
Why skip this file? It's no different from any other model file, right?

Reply:
This file imports all modules for models and every line is therefore an unused import. It's simpler to just skip the whole file instead of skipping each file manually.